src/components/templates/PartyMap/components/Map/hooks/useMapGrid.tsx

```diff
@@ -38,8 +39,8 @@ export const useMapGrid: ReactHook<UseMapGrid, UseMapGridReturn> = ({
             const column = colIndex + 1; // TODO: do these need to be here, can we zero index?
             const row = rowIndex + 1; // TODO: do these need to be here, can we zero index?
 
-            const seatedPartygoer = partygoersBySeat?.[row]?.[column] ?? null;
-            const hasSeatedPartygoer = !!seatedPartygoer;
+            const seatedPartygoer = partygoersBySeat?.[row]?.[column];
+            const hasSeatedPartygoer = seatedPartygoer !== undefined;
```

Comment:
What is the reason for changing it from `!!seatedPartygoer` to `eatedPartygoer !== undefined`?

Reply:
A general preference towards being more explicit in our intent with things, as that reduces the cognitive effort required to reason about it later. Potentially also a scope narrowing thing.

I don’t think I feel particularly strongly about this particular case either way, and I believe this particular field may also be able to be ‘null’ as well (as much as I dislike that it can be), so this change may not be accurate anyway.

Reply:
Looking at the typing here, that doesn't seem to be any need for `null` to exist? So will probably leave this as is unless there's a good use case why we shouldn't somewhere (eg. the typing lying to us)

![Screen Shot 2021-01-27 at 6 58 06 pm](https://user-images.githubusercontent.com/753891/105960624-9ccc5400-60d1-11eb-9ad0-790393b1c2ba.png)